README.md

```diff
@@ -1,20 +1,42 @@
 # Explorer visualization widget for the Jupyter notebook
 
-The `<urth-viz-explorer>` tag, based on the jupyter-incubator/declarativewidgets project, binds data to a visualization
-and provides UI controls to explore the data set.
+The [`<urth-viz-explorer>`](https://jupyter-incubator.github.io/declarativewidgets_explorer/#urth-viz-explorer) tag, based on the [jupyter-incubator/declarativewidgets project](https://github.com/jupyter-incubator/declarativewidgets),
+binds a dataframe to a visualization and provides UI controls to explore the data.
 
-The data abstraction is a dataframe, passed in either with the name of a kernel variable as the `ref` attribute,
-or using the [`<urth-core-dataframe>`](http://jupyter-incubator.github.io/declarativewidgets/docs.html#urth-core-dataframe) abstraction
-as a child element, wrapped in an `<urth-viz-query>` element to provide the ability to do interactive queries of the data set
-on the kernel.
+The dataframe is passed in either with the name of a kernel variable as the `ref` attribute,
+or embedded with a [`<urth-core-dataframe>`](http://jupyter-incubator.github.io/declarativewidgets/docs.html#urth-core-dataframe) as a child element, wrapped in an [`<urth-viz-query>`](https://jupyter-incubator.github.io/declarativewidgets_explorer/#urth-viz-query) to enable interactive queries executed on the kernel.
```

Comment:
I think the basic functions are more important than the structure (especially since most will probably use the explore function) so maybe we can expand on the functionality in the introductory paragraph and leave the technical details of how to wire up the data for the usage section below.